bundles/org.smarthomej.binding.mail/src/main/resources/OH-INF/i18n/mail.properties

```
@@ -16,3 +19,6 @@ sendHTMLAttachmentMessageActionDescription = Sends a HTML mail with an URL attac
 
 sendHTMLAttachmentsMessageActionLabel = send a HTML mail with several attachments
 sendHTMLAttachmentsMessageActionDescription = Sends a HTML mail with several URL attachments.
+
+sendComplexHtmlMessageActionLabel = send a html mail with several attachments and/or additional headers
```

```suggestion
sendComplexHtmlMessageActionLabel = send an HTML mail with several attachments and/or additional headers
```

In general we should fix the spelling in this file `a HTML` to `an HTML`.